doc/md/CLI/welcome.md

```diff
@@ -0,0 +1,40 @@
+---
+id: Welcome to Atlas
+slug: /CLI/
+sidebar_position: 2
+---
+
+## CLI
+
+Work with any data source from the command line.
+
+![Alt Text](https://release.ariga.io/images/assets/atlas-intro.gif)
+
+## Installation
+
+Download from one of our links or using curl.
+```shell
+curl -LO https://release.ariga.io/atlas/atlas-{os}-{arch}-{version}
+```
+Make the atlas binary executable.
+```shell
+chmod +x ./atlas-{os}-{arch}-{version}
+```
+Move the atlas binary to a file location on your system PATH.
+
+Example for Mac:
+```shell
+sudo mv ./atlas-darwin-{arch}-{version} /usr/local/bin/atlas
+sudo chown root: /usr/local/bin/atlas
+```
+Example for Linux:
+```shell
+sudo install -o root -g root -m 0755 ./atlas-linux-{arch}-{version} /usr/local/bin/atlas
```

Comment:
Curious (I don't know much about atlas yet), why are sudo right needed?

Reply:
just for installing in that folder, isn't that always?

Reply:
And why are you not using `go install`? It makes the executable available as well, doesn't it?

Reply:
I don't want this CLI to be "affiliated" with gophers but rather anyone who uses databases.